src/main/java/org/sagebionetworks/bridge/sdk/BridgeResearcherClient.java

```diff
@@ -42,14 +42,19 @@ public StudyParticipant getStudyParticipant(String email) {
     }
     
     @Override
-    public void updateStudyParticipant(String email, StudyParticipant participant) {
+    public void createStudyParticipant(StudyParticipant participant) {
+        session.checkSignedIn();
+        checkNotNull(participant);
+        
+        post(config.getParticipantsApi(), participant);
+    }
+    
+    @Override
+    public void updateStudyParticipant(StudyParticipant participant) {
```

Comment:
This is a breaking change to updateStudyParticipant(). Do we want to cut a new minor version of the JavaSDK?